src/test/java/pages/pompages/CheckboxPomPage.java

```diff
@@ -0,0 +1,31 @@
+package pages.pompages;
+
+import Constants.URLS;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class CheckboxPomPage extends BasePomPage {
+    private By CHECKBOXES_LINK = By.linkText("Checkboxes");
```

Comment:
This element belongs to navigation page, let's create another page for these navigation elements or let's use open page methods